pkg/k8s/client.go

```diff
@@ -61,6 +61,10 @@ func (s *LazyClient) Status() client.StatusWriter {
 	return s.getClient().Status()
 }
 
+func (s *LazyClient) SubResource(subResource string) client.SubResourceClient {
```

Comment:
This is just a side effect of satisfying the updated interface in controller-runtime v0.14.6